### PR TITLE
Added allsearch api url

### PR DESCRIPTION
**Pull request by rladdusaw**
Added a required config for pulibrary/allsearch_frontend#41

### CODE REVIEW

File: group_vars/allsearch_frontend/production.yml [hunk 1]
```diff
@@ -1,3 +1,4 @@
 ---
+allsearch_api_url: https://allsearch-api-production.princeton.edu
```
Comment (sandbergja) [inline group_vars/allsearch_frontend/production.yml:2]:
```suggestion
allsearch_api_url: https://allsearch-api.princeton.edu
```

The api url doesn't have `-production`

**Review by sandbergja — CHANGES_REQUESTED** [review]
Thanks, @rladdusaw -- just a small correction to the prod url

File: group_vars/allsearch_frontend/production.yml (hunk 1 above)
Comment (rladdusaw) [reply group_vars/allsearch_frontend/production.yml]: Thanks!

**Review by sandbergja — APPROVED** [review]
🐈‍⬛

**Pull request merged**